docs/commands/build.md

```diff
@@ -17,4 +17,6 @@ OPTIONS:
         --config <CONFIG>    The app configuration file [default: lenra.yml]
         --expose <EXPOSE>    Exposes services ports [possible values: app, devtool, postgres, mongo]
     -h, --help               Print help information
+        --production         Remove debug access to the app
```

Comment:
Maybe we must document here the alias `--prod` ?

Reply:
Seems to be a clap problem